projects/step-frontend/src/lib/modules/scheduler/execution-task-parameters-list/execution-task-parameters-list.component.html

```diff
@@ -0,0 +1,93 @@
+<section class="users-header">
+  <step-btn-group>
+    <button
+      type="button"
+      mat-raised-button
+      color="primary"
+      [disabled]="!('user-write' | hasRight | async)"
+      (click)="createParameter()"
+    >
+      New parameter
+    </button>
```

Comment:
We have a new way to do the primary add button as introduced in my table redesign PR.

`<button
      type="button"
      **mat-icon-button**
      **class="primary"**
      **matTooltip="new task"**
      **matTooltipPosition="below"**
      [disabled]="!('user-write' | hasRight | async)"
      (click)="createParameter()"
    >
      **<mat-icon>add</mat-icon>**
    </button>`

Take note that many properties changed